### ISSUE TITLE
Block groups/frames

### ISSUE THREAD

**Issue by TheShovel** [ScratchAddons/ScratchAddons#6390]
### Why this would be helpful

I work on projects with a lot of blocks, and it's really hard to organize them sometimes. This would help people with large projects be able to properly name multiple threads of code.

### How this addon works

Like the Gandi IDE block frames.
You can create a frame, start putting blocks into it and then give it a name. When you move that frame, all the blocks inside it will move together with the frame.

### Additional context

Here is how that looks like in Gandi.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/68913917/fe52dcec-b50a-4177-aa49-699eaa272343)

**Comment by DNin01** [comment]
It could just be a challenge to find a way to make sure the data-consisting comment isn't too cryptic or obscuring any scripts for users without this addon.

**Comment by TheShovel** [comment]
I don't think that would be an issue. You can change the size of the comment to be really small.

**Comment by CST1229** [comment]
> I don't think that would be an issue. You can change the size of the comment to be really small.

Yea, it could be collapsed, too.
![image](https://github.com/ScratchAddons/ScratchAddons/assets/68464103/398bf4c8-0677-40cc-bac0-b633fc48171b)
![image](https://github.com/ScratchAddons/ScratchAddons/assets/68464103/6ec32903-7217-4f07-968a-a3c9ac1416a3)

**Comment by DNin01** [comment]
Yeah, definitely collapsed.
